### ISSUE TITLE
[STEALTH] Broken by OWN DoT Potion

### ISSUE THREAD

**Issue by Vancerus** [WarEmu/WarBugs#14565]
**Expected behavior and actual behavior:**
When hitting an enemy with my own DoT potion, MY own stealth gets broken.
Message shows: "EnemyXYZ´s DoT potion has broken your stealth.

**Steps to reproduce the problem:**
Hit someone with a DoT potion and try to stealth.

**Testing Screenshots/Videos/Evidences (always needed):**
https://photos.app.goo.gl/KuuZcJeYvwhrqKHS9

**Comment by sioding** [comment]
So you are attacking someone with a dot .... well - all good. If you want a change for it use our suggestion forum.

**Comment by Vancerus** [comment]
what? When I am attacking someone with a DoT from an ability and afterwards go into stealth and it still ticks, it is supposted to break my stealth?!

**Comment by sioding** [comment]
Try to get a dev to check if its intended or a bug

**Comment by grunbag** [comment]
dot break stealth afaik

**Comment by Vancerus** [comment]
> dot break stealth afaik

No it does not when outgoing dmg (I apply DoT on enemy). 

Seems logical not to, that is the whole idea behind a Damage Over Time, to NOT be needed to do anything for the enemy to get dmg.

Video: https://photos.app.goo.gl/QJDFRJXd9cz3DMLn6

**Comment by grunbag** [comment]
On the vidéo it doesn’t break your stealth

**Comment by Vancerus** [comment]
> On the vidéo it doesn’t break your stealth

Exactly. Please check the first video in OP for the wrong behaviour with DoT Potions.

**Comment by grunbag** [comment]
which dot potion?

**Comment by Vancerus** [comment]
> which dot potion?

All of them. But in this particular case (see first video) it was the lvl 38 Lasting Scorching Blast Potion.
